packages/babel-plugin-import-jsx-pragma/package.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "@wordpress/babel-plugin-import-jsx-pragma",
 	"version": "1.0.1",
-	"description": "Babel transform plugin for automatically injecting an import to be used as the pragma for the React JSX Transform plugin.",
```

Comment:
Interesting, I'd have thought sentence-case would be better for those. Some descriptions are surely multi-sentence?

It's split on NPM (React seems to be all without periods; lodash all with, etc.)

Given we're in the "comments must have full-stops" camp (I like it), maybe we should be here too? 🤷‍♂️

Reply:
> Given we're in the "comments must have full-stops" camp (I like it), maybe we should be here too?

I would be also happy to see this change.

Reply:
Seems like something that would be good to have (though from what I can tell, not yet offered as a feature) from [`npm-package-json-lint`](https://github.com/tclindner/npm-package-json-lint), which we're already using and [have a predefined configuration](https://github.com/WordPress/gutenberg/tree/master/packages/npm-package-json-lint-config) for.

Reply:
Hey! I just released v3.3.0 of [`npm-package-json-lint`](
https://github.com/tclindner/npm-package-json-lint) with a new [`description-format`](https://github.com/tclindner/npm-package-json-lint/wiki/description-format) rule. It has two options:

1. `requireCapitalFirstLetter` - Throws an error if the first character in the description isn't capitalized.
2. `requireEndingPeriod` - Throws an error if the description doesn't end with a period.

Hopefully this works well for your team!

Reply:
Thanks, opened PR to start using this new rule: https://github.com/WordPress/gutenberg/pull/8596.